twoliter/embedded/build.Dockerfile

```diff
@@ -65,46 +65,53 @@ RUN \
    && cat ${PACKAGE}.spec >> rpmbuild/SPECS/${PACKAGE}.spec \
    && sed -i "/^%package\s.*$/a Epoch: ${BUILD_EPOCH}" rpmbuild/SPECS/${PACKAGE}.spec \
    && sed -i "s;\([Requires|Conflicts|Obsoletes]:.*[=>,>,<,<=,=]\) \(\%{version}\-\%{release}$\);\1 ${BUILD_EPOCH}:\2;" rpmbuild/SPECS/${PACKAGE}.spec \
-   && find . -maxdepth 1 -not -path '*/\.*' -type f -exec mv {} rpmbuild/SOURCES/ \; \
    && echo ${NOCACHE}
 
 USER root
+ARG BYPASS_SOCKET
 RUN --mount=target=/host \
-    find /host/build/rpms/ -mindepth 1 -maxdepth 1 -name '*.rpm' -size +0c -print -exec \
+    /host/build/tools/pipesys link --fd-socket "${BYPASS_SOCKET}" --target /bypass && \
+    find /bypass/build/rpms/ -mindepth 1 -maxdepth 1 -name '*.rpm' -size +0c -print -exec \
       ln -snft ./rpmbuild/RPMS {} \+ && \
     for pkg in ${PACKAGE_DEPENDENCIES} ; do \
-      [ -d "/host/build/rpms/${pkg}" ] || continue ; \
-      find /host/build/rpms/${pkg}/ -mindepth 1 -maxdepth 1 -name '*.rpm' -size +0c -print -exec \
+      [ -d "/bypass/build/rpms/${pkg}" ] || continue ; \
+      find /bypass/build/rpms/${pkg}/ -mindepth 1 -maxdepth 1 -name '*.rpm' -size +0c -print -exec \
         ln -snft ./rpmbuild/RPMS {} \+ ; \
     done && \
     createrepo_c \
-        -o ./rpmbuild/RPMS \
```

Comment:
nit: but making the diff bigger than it needs to be is probably not the best in a complicated PR like this. I would suggest re-format the Dockerfile in a subsequent PR.